diff --git a/node_modules/discord.js/typings/index.d.ts b/node_modules/discord.js/typings/index.d.ts
index b6adc89..2c8dbaf 100644
--- a/node_modules/discord.js/typings/index.d.ts
+++ b/node_modules/discord.js/typings/index.d.ts
@@ -534,6 +534,12 @@ export abstract class Channel extends Base {
 export type If<T extends boolean, A, B = null> = T extends true ? A : T extends false ? B : A | B;
 
 export class Client<Ready extends boolean = boolean> extends BaseClient {
+  log: (content: any, logType: any, client: any) => Promise<any>;
+    config: { prefix: string; token: string; channels: { messageLog: string; mainLog: string; }; colors: { join: string; leave: string; memberUpdate: string; roleCreate: string; roleDelete: string; roleUpdate: string; messages: { edit: string; delete: string; bulkDelete: string; reactionAdd: string; reactionRemove: string; }; }; events: { guildMemberAdd: boolean; guildMemberRemove: boolean; guildMemberUpdate: boolean; messageCreate: boolean; messageDelete: boolean; messageDeleteBulk: boolean; messageReactionAdd: boolean; messageReactionDelete: boolean; messageUpdate: boolean; ready: boolean; roleCreate: boolean; roleDelete: boolean; roleUpdate: boolean; }; };
+    hexToRGB: (hexCode: string) => readonly [number, number, number];
+    logger: any;
+    db: any;
+    logGuild: number;
   public constructor(options: ClientOptions);
   private actions: unknown;
   private presence: ClientPresence;
